internal/cmd/argus/scrape-configs/create/create.go

```diff
@@ -0,0 +1,164 @@
+			"If no payload is provided, a default payload will be used.",
+			"See https://docs.api.stackit.cloud/documentation/argus/version/v1#tag/scrape-config/operation/v1_projects_instances_scrapeconfigs_create for information regarding the payload structure.",
+		),
+		Args: args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`Create a Scrape Config job using default configuration`,
+				"$ stackit argus scrape-configs create"),
+			examples.NewExample(
+				`Create a Scrape Config job using an API payload sourced from the file "./payload.json"`,
+				"$ stackit argus scrape-configs create --payload @./payload.json"),
+			examples.NewExample(
+				`Create a Scrape Config job using an API payload provided as a JSON string`,
+				`$ stackit argus scrape-configs create --payload "{...}"`),
+			examples.NewExample(
+				`Generate a payload with default values, and adapt it with custom values for the different configuration options`,
+				`$ stackit argus scrape-configs generate-payload > ./payload.json`,
+				`<Modify payload in file, if needed>`,
+				`$ stackit argus scrape-configs create --payload @./payload.json`),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			instanceLabel, err := argusUtils.GetInstanceName(ctx, apiClient, model.InstanceId, model.ProjectId)
+			if err != nil {
+				instanceLabel = model.InstanceId
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to create a Scrape Config job on Argus instance %q?", instanceLabel)
+				err = p.PromptForConfirmation(prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Fill in default payload, if needed
+			if model.Payload == nil {
+				defaultPayload := argusUtils.DefaultCreateScrapeConfigPayload
+				if err != nil {
+					return fmt.Errorf("get default payload: %w", err)
+				}
+				model.Payload = &defaultPayload
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			_, err = req.Execute()
+			if err != nil {
+				return fmt.Errorf("create Scrape Config job: %w", err)
+			}
+
+			jobName := model.Payload.JobName
+
+			// Wait for async operation, if async mode not enabled
+			if !model.Async {
+				s := spinner.New(p)
+				s.Start("Creating scrape config")
+				_, err = wait.CreateScrapeConfigWaitHandler(ctx, apiClient, model.InstanceId, *jobName, model.ProjectId).WaitWithContext(ctx)
+				if err != nil {
+					return fmt.Errorf("wait for Scrape Config job creation: %w", err)
+				}
+				s.Stop()
+			}
+
+			operationState := "Created"
+			if model.Async {
+				operationState = "Triggered creation of"
+			}
+			p.Outputf("%s Scrape Configuration for Argus instance %q, with job name %q\n", operationState, instanceLabel, *jobName)
```

Comment:
Also for consistency with other descriptions, maybe rephrasing a bit could help readibility?

```suggestion
			p.Outputf("%s Scrape Config job with name %q for Argus instance %q\n", operationState, *jobName, instanceLabel)
```